src/include/sof/audio/module_adapter/module/generic.h

```diff
@@ -209,6 +210,10 @@ struct processing_module {
 
 	/* table containing the list of connected sources */
 	struct module_source_info *source_info;
+
+	/* total processed data after stream started */
+	uint64_t total_data_consumed;
+	uint64_t total_data_produced;
```

Comment:
Not blocking: just wondering whether 64bit is really needed and worth the memory/cycles spent. I guess the any code using get_total_process_data anyways needs to handle possibility of wrap-arounds, so not sure 64bit is needed (latency will never exceed 32bit range).

But alas, this is not really related to this PR as the data type was picked already for copier.